instance/management/commands/activity_csv.py

```diff
@@ -123,9 +125,13 @@ def handle(self, *args, **options):  # pylint: disable=too-many-locals
 
                 instance_age = datetime.now(instance.created.tzinfo) - instance.created
 
-                emails = [user.email for user in instance.lms_users.all()]
+                try:
+                    email = instance.betatestapplication_set.get().user.email
+                except BetaTestApplication.DoesNotExist:
+                    email = 'N/A'
```

Comment:
That would also be a good improvement, but probably overkill for this iteration - all the betatesters we are interested in should have this email field set.